packages/grid/test/drag-and-drop.common.js

```diff
@@ -1073,5 +1068,20 @@ describe('drag and drop', () => {
       fireDragOver(grid.__getViewportRows()[0], 'above');
       expect(grid.$.table.scrollTop).to.be.within(scrollTop - 200, scrollTop - 100);
     });
+
+    it('should add/remove drag-source- part when scrolling', () => {
+      grid.rowsDraggable = true;
+      grid.selectItem(grid.items[0]);
+      fireDragStart();
+
+      // Scroll down so that the drag source cell leaves the viewport
+      grid.scrollToIndex(50);
+      // Expect no cells with drag-source-row-cell part in the DOM
+      expect(grid.$.items.querySelector('tr:not([hidden]) [part~="drag-source-row-cell"]')).to.be.null;
+
+      grid.scrollToIndex(0);
+      expect(getFirstCell(grid).getAttribute('part')).to.contain('drag-source-row-cell');
+      grid.rowsDraggable = false;
```

Comment:
nit: this is unnecessary since a fresh new grid instance is created for each test

Reply:
You mean the very last line? I can remove that if you it's unnecessary

Reply:
Yes, the last line which resets the `rowsDraggable`